test/ui-testing/publishButton.test.js

```diff
@@ -5,10 +5,18 @@ beforeAll(async () => {
   await page.goto('file://' + path, {waitUntil: 'domcontentloaded'});
 });
 
+global.window = window
```

Comment:
Hmm now I see: `    ReferenceError: window is not defined` so will try removing this line?

Reply:
```suggestion
# global.window = window
```